content/docs/command-reference/remote/modify.md

```diff
@@ -758,6 +758,44 @@ more information.
 
 </details>
 
+<details>
+
+### Click for OSF
+
+- `url` - remote location:
+
+  ```dvc
+  $ dvc remote modify myremote url \
+        osf://odf.io/path/to/dir
+
+- `project` - [OSF](https://osf.io/) project name. You can see it in the `url` line
+  of your project page.
+
+  ```dvc
+  $ dvc remote modify myremote project <project_name>
+  
+- `osf_username` - username (e-mail) that you specify during registration.
+  ```dvc
+  $ dvc remote modify myremote osf_username \
+        example@mail.com
+  ```
+- 'password' - password for you osf account.
+  ```dvc
+  $ dvc remote modify osfremote --local password 12345
+  ```
+
+> The username and password (may) contain sensitive user info.
+> Therefore, it's safer to add them with the `--local` option, so they're
+> written to a Git-ignored config file.
+  
+  Also it is possible to specify password by setting
+  envirement variable `OSF_PASSWORD`. Run
+  ```$ export OSF_PASSWORD=12345```
```

Comment:
This should be part of the password bullet, maybe after the code block (on that same indentation level).

Reply:
Can the other parameters (project and username) also be set this way maybe?

Reply:
Fixed.

Reply:
Yes, I think it is a good idea to let `user` and `project` be set through the environment variables.